main.go

```diff
@@ -108,7 +109,10 @@ func main() {
 	}
 
 	if versionFlag {
-		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH)
+		cmd := exec.Command("git", "--version")
+		stdout, _ := cmd.Output()
+		git_version := stdout[12:]
+		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s, git version = %s \n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH, git_version)
```

Comment:
```suggestion
		cmd := secureexec.Command("git", "--version")
		stdout, _ := cmd.Output()
		git_version := strings.Trim(strings.TrimPrefix(string(stdout), "git version "), " \r\n")
		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s, git version=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH, git_version)
```

Reply:
I couldn't use secureexec as it generating some errors.

Reply:
Which ones?

Reply:
That's because you didn't import it.

Reply:
I don't know lot about secure but I tried to import it and it didn't worked. I cloned go's text repo but it didn't worked.

Reply:
I couldn't find anything related to secureexec on internet. do you have some resource where I can look it up please.

Reply:
secureexec is a local package in lazygit.

Reply:
```diff
diff --git a/main.go b/main.go
index 405b384f..59b191fe 100644
--- a/main.go
+++ b/main.go
@@ -5,7 +5,6 @@ import (
 	"fmt"
 	"log"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"runtime"
 	"runtime/debug"
@@ -18,6 +17,7 @@ import (
 	"github.com/jesseduffield/lazygit/pkg/env"
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 	"github.com/jesseduffield/lazygit/pkg/logs"
+	"github.com/jesseduffield/lazygit/pkg/secureexec"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	yaml "github.com/jesseduffield/yaml"
 	"github.com/samber/lo"
@@ -109,7 +109,7 @@ func main() {
 	}
 
 	if versionFlag {
-		cmd := exec.Command("git", "--version")
+		cmd := secureexec.Command("git", "--version")
 		stdout, _ := cmd.Output()
 		git_version := strings.Trim(strings.TrimPrefix(string(stdout), "git version "), " \r\n")
 		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s, git version=%s \n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH, git_version)
```

Reply:
thanks for explaining. I just made a commit with these changes. please let me know if there any more work to do.